rules/tailwind.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+  extends: [
+    "stylelint-config-sass-guidelines",
+    "stylelint-config-rational-order",
+  ],
+  plugins: [
+    "stylelint-scss",
+    "stylelint-order",
+    "stylelint-config-rational-order/plugin",
+  ],
+  rules: {
+    /* Specify the alphabetical order of properties within declaration blocks. */
+    /* https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-alphabetical-order */
+    "order/properties-alphabetical-order": null,
```

Comment:
Please do not re-introduce rules from core. This file should only extend the core configuration. The usage in the stylelint configuration should look like:

```
{
  "extends": [
    "@edenspiekermann/stylelint-config",
    "@edenspiekermann/stylelint-config/tailwind"
  ]
}
```

or

```
{
  "extends": [
    "@edenspiekermann/stylelint-config",
    "@edenspiekermann/stylelint-config/sass",
    "@edenspiekermann/stylelint-config/tailwind"
  ]
}
```